src/swap/components/SwapSettingsSlippageLayoutBottomSheet.tsx

```diff
@@ -0,0 +1,62 @@
+import { Children, cloneElement, isValidElement, useMemo } from 'react';
+import { findComponent } from '../../internal/utils/findComponent';
+import { cn } from '../../styles/theme';
+import type { SwapSettingsSlippageLayoutReact } from '../types';
+import { SwapSettingsSlippageDescription } from './SwapSettingsSlippageDescription';
+import { SwapSettingsSlippageInput } from './SwapSettingsSlippageInput';
+import { SwapSettingsSlippageTitle } from './SwapSettingsSlippageTitle';
+import { SwapSettingsSlippageToggle } from './SwapSettingsSlippageToggle';
+
+export function SwapSettingsSlippageLayoutBottomSheet({
+  children,
+  className,
+  customSlippageEnabled,
+  onToggleCustomSlippage,
```

Comment:
listeners are not allowed in any componet, the only listeners allowed are the one controlled by the lifecyclestatus.

Reply:
Able to remove these listeners with the refactor

Reply:
Will start to think Life Cycle Status first